src/utils/auth.tsx

```diff
@@ -0,0 +1,31 @@
+import { NextPage } from "next";
+import { useSession, getSession } from "next-auth/react";
+import { useRouter } from "next/router";
+import Spinner from "../components/Spinner";
+
+
+/**
```

Comment:
Actually I take back the other comment, maybe we should add something in the JSDoc here for reference in future pages (so we know the arrow notation is necessary)? At least one or the other should be added though.